src/check_env.sh

```diff
@@ -22,12 +22,12 @@ command_exists docker-machine "$DOCKER_MACHINE_BIN" https://docs.docker.com/mach
 command_exists watchmedo "$WATCHMEDO_BIN" https://github.com/gorakhargosh/watchdog#installation
 command_exists rsync "$RSYNC_BIN" ""
 
-if ! (bash --version | grep "version 4" >/dev/null); then
-    echo "check bash(version 4)...\e[31mnot found\e[m"
+if ! (bash --version | grep -e "version 4" -e "version 5" >/dev/null); then
```

Comment:
```suggestion
if ! (bash --version | grep -e "version [45]" >/dev/null); then
```
`grep -e` (正規表現) を使うなら、上記のようにまとめてしまった方がシンプルですね。